<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="activitiesList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Activity</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Start time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>End Time</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addActivity">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Activity...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeActivity">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Activity...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>394</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dateEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>dateEdit</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
